Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -71,14 +71,34 @@ public class ResourcesProjectMapper: ProjectMapping {
             sideEffects.append(sideEffect)
         }
 
+        if target.supportsSources && target.containsObjcFiles {
+            let (headerFilePath, headerData) = synthesizedObjcHeaderFile(bundleName: bundleName, target: target, project: project)
+
+            let headerHash = try headerData.map(contentHasher.hash)
+            let headerFile = SourceFile(path: headerFilePath, contentHash: headerHash)
+            let headerSideEffect = SideEffectDescriptor.file(.init(path: headerFilePath, contents: headerData, state: .present))
+
+            // FIXME: Is there any better way than GCC_PREFIX_HEADER? And what if the target would have set this already, then we're breakings something here.
+            var settings = modifiedTarget.settings?.base ?? SettingsDictionary()
+            settings["GCC_PREFIX_HEADER"] = .string(headerFile.path.url.relativePath)
+            modifiedTarget.settings = modifiedTarget.settings?.with(base: settings)
```

Comment:
This needs to be discussed, there's hopefully a better way, shouldn't stay like this.